jira/github_jira.py

```diff
@@ -62,6 +62,17 @@
     if not any([bug.html_url in jira_obj.fields.description for jira_obj in jira_issues]):
         to_create.append(bug)
 
+dod_title = "Definition of Done"
+dod_content = """
+Code Quality: Code adheres to team coding standards and is well-documented.
+Testing: All tests pass successfully, and relevant tests are updated.
+Review and Approval: PR has been reviewed, and feedback has been addressed.
+Documentation: Relevant documentation has been updated (e.g., README, release notes).
+Functionality: Code meets requirements and works as expected without regressions.
+Merging: PR is up to date with the base branch and free of merge conflicts.
+Backporting: Relevant backport labels are added based on the nature of the fix, and backported PRs are approved and merged.
```

Comment:
```suggestion
* Code Quality: Code adheres to team coding standards and is well-documented.
* Testing: All tests pass successfully, and relevant tests are updated.
* Review and Approval: PR has been reviewed, and feedback has been addressed.
* Documentation: Relevant documentation has been updated (e.g., README, release notes).
* Functionality: Code meets requirements and works as expected without regressions.
* Merging: PR is up to date with the base branch and free of merge conflicts.
* Backporting: Relevant backport labels are added based on the nature of the fix, and backported PRs are approved and merged.
```
I think this will be easier to read in the ticket if it's a bulleted list.